### PR TITLE
Packagist: Fix link bug, inconsistencies & improve UI

### PR DESCRIPTION
**Pull request by mr-max**
**Bug fixed:** 
- Wrong sourceUrl when query contains spaces

**Enhancements:**
- The UI was improved and simplified by using the `basic1` variant
- Now unused custom CSS and handlebars files could be removed
- For consistency, the favers number is now _commified_
- For consistency, a star icon is used for GitHub stars

**Before:**
image

**After**
image

---
Instant Answer Page: https://duck.co/ia/view/packagist
Maintainer: @somus

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mr-max Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/spice/packagist/footer.handlebars [hunk 1]
```diff
@@ -1,3 +1 @@
-<div class="tile__update__info one-line">
-    <span class="ddgsi">♥</span> {{favers}} &bullet; {{downloads}} Downloads
-</div>
\ No newline at end of file
+<span class="sep--after text--secondary"><i class="ddgsi ddgsi-download"></i> {{downloads}}</span><a href="{{repository}}" title="View on GitHub" class="text--secondary"><i class="ddgsi ddgsi-star"></i> {{favers}}</a>
```
Comment (moollaza) [inline share/spice/packagist/footer.handlebars]: Let's keep this nested for readability sake

**Comment by moollaza** [comment]
@mr-max thanks for getting rid of all that CSS and correcting the sourceURL.

It looks like some of the descriptions in the Instant Answer are big enough to warrant a taller tile though. I think these changes make the tile too short. I would suggest you copy the design of the RubyGems IA: https://duckduckgo.com/?q=rubygems+math&ia=software.

For the footer, I would either stick with the dot separator because we never use the vertical sep in the footers. Alternatively, we can float the downloads and likes to the left and right sides of the footer like we do for the SublimePackages IA: https://duckduckgo.com/?q=sublimetext+package+code&ia=software

**Comment by mr-max** [comment]
image

@moollaza: I incorporated your feedback: The tiles are now larger and use bullets as dividers

**Comment by edgesince84** [comment]
@mr-max,  thanks again for fixing the Packagist IA! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in UI/UX improvements.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to UI/UX

OR

- Test live <topic> Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by moollaza** [comment]
@mr-max Thanks! This LGTM :+1:

**Pull request merged**